index.html

```diff
@@ -21,6 +21,11 @@
                 <div>
                     <i class="fa-solid fa-bars menu-bar"></i>
                 </div>
+                <ul class="nav-links">
+                    <li class="port1"><a href="#portfolio">Portfolio </a></li>
+                    <li class="ab1"><a href="#about">About </a></li>
+                    <li class="cont1"><a href="#contact">Contact</a></li>
+                </ul>
 
```

Comment:
- [ ] I think while making the changes some bugs were introduced, currently your header section is going out of the viewport, please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/206867594-7e5fbe95-cc53-4756-bef4-69262662c1ab.png)

I think we can fix this by removing the excessive padding and margin from the logo and using` justify-content: space around ` in the nav. you don't need to specify flex-direction row since by default it is already set to row.
```
logo {
    padding: 25px 620px 0 155px;
  }
```
```
nav {
  display: flex;
  justify-content: space-around;
}
```